src/router/index.js

```diff
@@ -44,7 +44,10 @@ export default {
   registries: {
     overview: '/home/registries/overview',
     companies: '/home/registries/companies',
-    companiesCRUD: '/home/registries/companiesCRUD',
+    companiesCRUD: {
```

Comment:
Lets drop the `CRUD`, it's weird, especially when it's user visible.
Maybe something like this?
```
  companies: {
    overview: '/home/registries/companies',
    create: '/home/registries/companies/create',
    edit: '/home/registries/companies/edit/:companyId',
}
```